pkg/core/errors.go

```diff
@@ -91,11 +91,13 @@ func (err *CompileError) Format(s fmt.State, verb rune) {
 			fmt.Fprint(s, "\n")
 			err.Annotation.Format(s, verb)
 			fmt.Fprintf(s, "\nin %s\n", err.File.Path())
-			fnode := &NodeContent{
-				Endpoints: err.Annotation.Node,
-				Content:   err.Annotation.Node.Content(),
+			if err.Annotation.Node != nil {
```

Comment:
The trailing newline that this condition would cause (from the Fprintf in the previous line) would be awkward wouldn't it?

Reply:
🤷‍♂️  I guess I can just move that newline inside the if block.